Samples/SwiftUITestbed/Sources/MainScreen.swift

```diff
@@ -0,0 +1,116 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import BlueprintUI
+import MarketUI
+import MarketWorkflowUI
+import ViewEnvironment
+
+struct MainScreen: MarketScreen {
+    enum Field: Hashable {
+        case title
+    }
+
+    @FocusState var focusedField: Field?
+
+    let title: String
+    let didChangeTitle: (String) -> Void
+
+    let allCapsToggleIsOn: Bool
+    let allCapsToggleIsEnabled: Bool
+    let didChangeAllCapsToggle: (Bool) -> Void
+
+    let didTapPushScreen: () -> Void
+    let didTapPresentScreen: () -> Void
+
+    let didTapClose: (() -> Void)?
+
+    func element(
+        in context: MarketWorkflowUI.MarketScreenContext,
+        with styles: MarketTheming.MarketStylesheet
+    ) -> BlueprintUI.Element {
+        MarketScreenContent {
+            Column(
+                underflow: .justifyToStart,
+                minimumSpacing: styles.spacings.spacing200
+            ) {
+                MarketInlineSectionHeader(
+                    style: styles.headers.inlineSection20,
+                    title: "Title"
+                )
+
+                MarketTextField(
+                    style: styles.fields.textField,
+                    label: "Text",
+                    text: title,
+                    onChange: didChangeTitle,
+                    onReturn: { _ in focusedField = nil }
+                )
+                .focused(when: $focusedField, equals: .title)
+                .onAppear { focusedField = .title }
+
+                Row(
+                    alignment: .center,
+                    minimumSpacing: styles.spacings.spacing200
+                ) {
+                    MarketLabel(
+                        textStyle: styles.typography.semibold20,
+                        color: styles.colors.text10,
+                        text: "All Caps"
+                    )
+
+                    MarketToggle(
+                        style: styles.toggle.normal,
+                        isOn: allCapsToggleIsOn,
+                        isEnabled: allCapsToggleIsEnabled,
+                        accessibilityLabel: "is all caps",
+                        onChange: didChangeAllCapsToggle
+                    )
+                }
+
+                Spacer(styles.spacings.spacing50)
+
+                MarketInlineSectionHeader(
+                    style: styles.headers.inlineSection20,
+                    title: "Navigation"
+                )
+
+                MarketButton(
+                    style: styles.button(rank: .secondary),
+                    text: "Push Screen",
+                    onTap: didTapPushScreen
+                )
+
+                MarketButton(
+                    style: styles.button(rank: .secondary),
+                    text: "Present Screen",
+                    onTap: didTapPresentScreen
+                )
```

Comment:
Nit: Might be nice to add a "Resign Focus" button here for testing
```suggestion
                )

                MarketButton(
                    style: styles.button(rank: .secondary),
                    text: "Resign Focus",
                    onTap: { focusedField = nil }
                )
```